.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,8 +1,7 @@
 ### Pull Request Checklist
 
-<!-- Please read CONTRIBUTING.rst before submitting your pull request -->
-
-* [ ] UI change has been tested on both light and dark themes
+* [ ] I read the [contributing guide](https://github.com/vector-im/element-ios/blob/documentation_update/CONTRIBUTING.md)
```

Comment:
Pointing to `documentation_update` branch right now, obviously as required. Adding this not to forget to change it to default branch in future(maybe just before merging this).